lib/rmt/scc_sync.rb

```diff
@@ -29,8 +43,18 @@ def sync
     end
 
     @logger.info('Done!')
+  rescue SUSE::Connect::Api::InvalidCredentialsError
+    @logger.error('SCC credentials not valid.')
+  rescue Interrupt
+    puts 'Aborting! You need to rerun this command to have a consistent state.'
```

Comment:
Maybe this should also be logged? (You can send SIGINT to a background process, even though it is unlikely)